.github/workflows/cicd-pipeline.yml

```
@@ -316,23 +311,63 @@ jobs:
+            
+              Este PR rodará os status checks. Assim que passarem, pode ser mergeado.`
+            });
 
       - name: Delete Merged Release or Hotfix Branch
         if: steps.create_tag.outputs.should_run == 'true'
+        uses: actions/checkout@v5
+        with:
+          ssh-key: ${{ secrets.DEPLOY_KEY_PRIVATE }}
+      - env:
+          BRANCH_TO_DELETE: ${{ github.event.pull_request.head.ref }}
         run: |
```

actionlint (1.7.8)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)